Python/sysmodule.c

```diff
@@ -3664,6 +3664,11 @@ _PySys_UpdateConfig(PyThreadState *tstate)
         COPY_LIST("path", config->module_search_paths);
     }
 
+#ifndef PyTraceBack_LIMIT
```

Comment:
Because `PyTraceBack_LIMIT` isn't defined in the header file, this's to solve the undefined problem. But if allowed, I would like to move `PyTraceBack_LIMIT` to `pycore_call.h`